src/utils/avaxMainnetTokenMapping.ts

```diff
@@ -1,15 +1,14 @@
 import { ZERO_ADDRESS } from 'constants/index'
 import { isAddress } from 'utils'
-import { AVAX_MAINNET_TOKEN_LIST } from 'constants/tokenLists/avax.mainnet.tokenlist'
 
 export const getAvaxMainnetTokenLogoURL = (address: string) => {
-  let uri = AVAX_MAINNET_TOKEN_LIST.tokens.find(item => item.address.toLowerCase() === address.toLowerCase())?.logoURI
+  let uri
```

Comment:
this will be affect token image, please check again careful

Reply:
This was moved to outside here
https://github.com/KyberNetwork/dmm-interface/pull/1034/files#diff-6147e3c1761df71fd40952dbcbac388a45f80b8c318f367ef41048351a2c0c99R479-R485